gluefactory/models/lines/pold2_extractor.py

```diff
@@ -23,7 +23,8 @@
 
 from gluefactory.models.base_model import BaseModel
 from gluefactory.models.lines.line_refinement import merge_lines_torch
-from gluefactory.models.lines.pold2_mlp import POLD2_MLP
+#from gluefactory.models.lines.pold2_mlp import POLD2_MLP
```

Comment:
Remark: we should remember makingtbis configurable (use of hardiks or your class) in case we end up considering both.
Please mind this when you integrate @hardik01shah

Reply:
Sounds good!